sdk/identity/azure-identity/src/default_azure_credential.cpp

```diff
@@ -10,13 +10,34 @@
 #include "private/chained_token_credential_impl.hpp"
 #include "private/identity_log.hpp"
 
+#include <azure/core/internal/environment.hpp>
+#include <azure/core/internal/strings.hpp>
+
 using namespace Azure::Identity;
 using namespace Azure::Core::Credentials;
 
 using Azure::Core::Context;
+using Azure::Core::_internal::Environment;
+using Azure::Core::_internal::StringExtensions;
 using Azure::Core::Diagnostics::Logger;
 using Azure::Identity::_detail::IdentityLog;
 
+namespace {
+std::string TrimString(std::string s)
```

Comment:
Don't we already have one of these?

Reply:
Thank you, I moved this into `StringExtensions`.